evcxr_repl/Cargo.toml

```diff
@@ -13,9 +13,10 @@ edition = "2018"
 
 [dependencies]
 evcxr = { version = "=0.4.7", path = "../evcxr" }
-rustyline = "5.0.2"
+rustyline = "6.0.0"
```

Comment:
Looks like rustyline 6 uses the `non_exhaustive` feature, which was only stabilised in 1.40.0. I think I'm OK with dropping support for older versions of rust so long as it gives us something. Can you update .travis.yml to bump the minimum supported version to 1.40.0?